### DISCUSSION TITLE
Can we also extract zotero's highlight and its colors?

### DISCUSSION THREAD

**Discussion by higalpe** [e-zz/logseq-pdf-extract#14] [Q&A] [answered]
Plugin running fine and quite happy with it. But now I would like to import also the annotations done on Zotero + the color of the highlight.

Could we configure this on the plugin? How could I add a Logseq emoji color that is similar to the zotero highlight colour from each annotation?

Thank you!

**Comment by e-zz** [comment, marked as answer]
Sorry I just saw your questions. 

Importing annotations from Zotero is not a problem.  So, in principle we could import them at least as plain text (or get Zotero links that lead to them. For area annotations, we could get at least the box coordinates easily).
It's just we (Logseq plugins) can't modify the Logseq PDF Annotation metafiles. So it's not possible yet to fully import those annotations as normal Logseq PDF annotations (or possibly with highlight color).

It's a pity to say so: I tried to circumvent the limitation and in th end I failed. But I will keep searching from time to time. Hope the Logseq dev team could release more plugin APIs in near future to make this happen.

**Reply by higalpe** [reply]
Ok, that is a pity. Hopefully in the future Logseq facilitates the implementation or a workaround is founded. Thank you for take into account this feature and have it in mind. And thank you for such a nice local solution 😁
